releng/org.dataflowanalysis.analysis.targetplatform/org.dataflowanalysis.analysis.targetplatform.targetplatform.target

```diff
@@ -109,6 +109,9 @@
       <unit id="org.junit.platform.suite.engine" version="0.0.0"/>
       <unit id="org.junit.vintage.engine" version="0.0.0"/>
       <unit id="org.apiguardian" version="0.0.0"/>
+			<unit id="com.fasterxml.jackson.core.jackson-annotations" version="0.0.0"/>
+			<unit id="com.fasterxml.jackson.core.jackson-core" version="0.0.0"/>
+			<unit id="com.fasterxml.jackson.core.jackson-databind" version="0.0.0"/>
     </location>
```

Comment:
Is this GitHub, or is the indentation here rather crazy?

Reply:
![Bildschirmfoto 2024-02-21 um 08 46 56](https://github.com/DataFlowAnalysis/DataFlowAnalysis/assets/131791812/1ae1b43d-20f1-4fe7-88dc-9eecd3a09240)
Git is at fault